### ISSUE TITLE
Fix `selector-pseudo-class-parentheses-space-inside` false positive for multiline pseudo-classes

### ISSUE THREAD

**Issue by splincode** [stylelint-stylistic/stylelint-stylistic#70]
### What minimal example or steps are needed to reproduce the bug?

appearance.less

```less
.appearance-invalid(@content) {
    &:is([data-mode~='invalid'], .tui-invalid, :invalid):not(
            [data-mode~='readonly'],
            [data-mode~='valid'],
            [data-state='disabled'],
            :disabled
        ) {
        @content();
    }
}
```

```
  52:8  ✖  Unexpected whitespace before ")"  @stylistic/selector-pseudo-class-parentheses-space-inside
```

Please, you should ignore `\n` before `)`, because prettier changed it from `...:disabled)` to `...:disabled\n\t)`
